iphone/CoreApi/CoreApi/Logger/Logger.mm

```diff
@@ -132,10 +115,54 @@ bool AssertMessage(base::SrcPoint const & src, std::string const & message)
 
 // MARK: - Private
 
-+ (BOOL)isFileLoggingEnabled {
-  return [self logger].fileHandle != nil;
++ (void)enableFileLogging {
+  Logger * logger = [self logger];
+  NSFileManager * fileManager = [NSFileManager defaultManager];
+
+  // Create a log file if it doesn't exist and setup file handle for writing.
+  NSString * filePath = [[fileManager temporaryDirectory] URLByAppendingPathComponent:kLogFileName].path;
+  if (![fileManager fileExistsAtPath:filePath]) {
+    [fileManager createFileAtPath:filePath contents:nil attributes:nil];
+  }
+  NSFileHandle * fileHandle = [NSFileHandle fileHandleForWritingAtPath:filePath];
+  if (fileHandle == nil) {
+    LOG(LERROR, ("Failed to open log file for writing"));
```

Comment:
Does it make sense to show the same alert to a user?

Reply:
```suggestion
    LOG(LERROR, ("Failed to open log file for writing", filePath.UTF8String));
```

Reply:
> Does it make sense to show the same alert to a user?

I'm not sure that we should notify user about this "under the hood" processes or errors. It will be hard to investigate a failure reason even if we show the alert...